Auto update failing for jp2a

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [jp2a](https://github.com/termux/termux-packages/blob/2d0f5335222d546aba5e46e3ac57472166e9a6a4/packages/jp2a) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating jp2a [Current version: 1.2.0]
INFO: package being updated to 1.3.0.
Downloading https://github.com/Talinx/jp2a/archive/v1.3.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  589k    0  589k    0     0   736k      0 --:--:-- --:--:-- --:--:--  736k

You are about to commit these changes:

--------------------
diff --git a/packages/jp2a/build.sh b/packages/jp2a/build.sh
index a234e2231..8b297ee92 100644
--- a/packages/jp2a/build.sh
+++ b/packages/jp2a/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://csl.name/jp2a/
 TERMUX_PKG_DESCRIPTION="A simple JPEG to ASCII converter"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.2.0"
+TERMUX_PKG_VERSION="1.3.0"
 TERMUX_PKG_SRCURL=https://github.com/Talinx/jp2a/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=a19088bb4ba27e8a5524f26f0f1622529ca087e36f5a060c6535c11b266106d1
+TERMUX_PKG_SHA256=68b9cefc291a9bc5b6210dc3cb29173e7222e49a3aec6cab51b0f5e5e696371f
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libcurl, libjpeg-turbo, libpng, ncurses"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/jp2a): 1.3.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package jp2a exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building jp2a for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  24378      0 --:--:-- --:--:-- --:--:-- 24412
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1518      0 --:--:-- --:--:-- --:--:--  1517
gpg: Signature made Sun 22 Sep 2024 05:55:00 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  23073      0 --:--:-- --:--:-- --:--:-- 23044
100 13320  100 13320    0     0  23068      0 --:--:-- --:--:-- --:--:-- 23044
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1561      0 --:--:-- --:--:-- --:--:--  1562
gpg: Signature made Fri 20 Sep 2024 12:44:57 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  23469      0 --:--:-- --:--:-- --:--:-- 23470
100 13120  100 13120    0     0  23465      0 --:--:-- --:--:-- --:--:-- 23428
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1433      0 --:--:-- --:--:-- --:--:--  1433
gpg: Signature made Sun 22 Sep 2024 09:00:03 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libnghttp2@1.63.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp2/libnghttp2_1.63.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 99048  100 99048    0     0  1581k      0 --:--:-- --:--:-- --:--:-- 1585k
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.5.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp3/libnghttp3_1.5.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63400  100 63400    0     0   990k      0 --:--:-- --:--:-- --:--:--  998k
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2024.07.02_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  122k  100  122k    0     0  1919k      0 --:--:-- --:--:-- --:--:-- 1942k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2024.07.02_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  111k  100  111k    0     0  1775k      0 --:--:-- --:--:-- --:--:-- 1803k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zlib/zlib_1.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 60516  100 60516    0     0   823k      0 --:--:-- --:--:-- --:--:--  820k
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.3.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2261k  100 2261k    0     0  22.4M      0 --:--:-- --:--:-- --:--:-- 22.5M
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.3.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  255k  100  255k    0     0  3418k      0 --:--:-- --:--:-- --:--:-- 3451k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libssh2/libssh2_1.11.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  193k  100  193k    0     0  2655k      0 --:--:-- --:--:-- --:--:-- 2683k
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.10.1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcurl/libcurl_8.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  884k  100  884k    0     0   9.8M      0 --:--:-- --:--:-- --:--:--  9.9M
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.10.1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/curl/curl_8.10.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  207k  100  207k    0     0  2650k      0 --:--:-- --:--:-- --:--:-- 2656k
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libjpeg-turbo@3.0.4 if necessary...
Found libjpeg-turbo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libj/libjpeg-turbo/libjpeg-turbo_3.0.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  365k  100  365k    0     0  4074k      0 --:--:-- --:--:-- --:--:-- 4105k
extracting libjpeg-turbo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libjpeg-turbo-progs@3.0.4 if necessary...
Found libjpeg-turbo-progs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libj/libjpeg-turbo-progs/libjpeg-turbo-progs_3.0.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 95488  100 95488    0     0  1490k      0 --:--:-- --:--:-- --:--:-- 1504k
extracting libjpeg-turbo-progs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.44 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpng/libpng_1.6.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  191k  100  191k    0     0  2654k      0 --:--:-- --:--:-- --:--:-- 2692k
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  524k  100  524k    0     0  6138k      0 --:--:-- --:--:-- --:--:-- 6171k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87768  100 87768    0     0  1253k      0 --:--:-- --:--:-- --:--:-- 1260k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33072  100 33072    0     0   576k      0 --:--:-- --:--:-- --:--:--  587k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39580  100 39580    0     0   519k      0 --:--:-- --:--:-- --:--:--  522k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/Talinx/jp2a/archive/v1.3.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  589k    0  589k    0     0   862k      0 --:--:-- --:--:-- --:--:--  862k
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /usr/bin/autoconf
autoreconf: running: /usr/bin/autoheader
autoreconf: running: automake --add-missing --copy --no-force
configure.ac:16: installing './compile'
configure.ac:9: installing './install-sh'
configure.ac:9: installing './missing'
Makefile.am: installing './INSTALL'
src/Makefile.am: installing './depcomp'
autoreconf: Leaving directory '.'
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
checking for aarch64-linux-android-doxygen... no
checking for doxygen... no
configure: WARNING: doxygen not found - will not generate any doxygen documentation
checking for aarch64-linux-android-perl... no
checking for perl... /usr/bin/perl
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-linux-android-strip... llvm-strip
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for aarch64-linux-android-gcc... aarch64-linux-android-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-linux-android-clang accepts -g... yes
checking for aarch64-linux-android-clang option to enable C11 features... none needed
checking whether aarch64-linux-android-clang understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-linux-android-clang... none
checking for jpeg_start_decompress in -ljpeg... yes
checking for png_create_read_struct in -lpng... yes
checking for WebPGetInfo in -lwebp... no
checking for exif_data_new_from_file in -lexif... no
checking for library containing floor... -lm
checking enable-curl... yes
checking for curl-config... /home/builder/.termux-build/jp2a/tmp/config-scripts/curl-config
checking curl-config --cflags... -I/data/data/com.termux/files/usr/include
checking curl-config --libs... -L/data/data/com.termux/files/usr/lib -lcurl
checking for curl_easy_init in -lcurl... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for vfork.h... no
checking for jpeglib.h... yes
checking for curl/curl.h... yes
checking for fcntl.h... yes
checking for curses.h... yes
checking for term.h... yes
checking for ncurses/term.h... yes
checking for math.h... yes
checking for limits.h... yes
checking for locale.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking for wchar.h... yes
checking for setjmp.h... yes
checking for tgetent in -lncurses... yes
checking pkg-config is at least version 0.9.0... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for C/C++ restrict keyword... __restrict__
checking for size_t... yes
checking for _Bool... yes
checking for stdbool.h that conforms to C99... yes
checking for pid_t... yes
checking for fork... yes
checking for vfork... yes
checking for working fork... cross
configure: WARNING: result yes guessed because of cross compilation
checking for working vfork... (cached) yes
checking for pid_t... (cached) yes
checking for memset... yes
checking for atexit... yes
checking for mbrlen... yes
checking for memset... (cached) yes
checking for setlocale... yes
checking for strerror... yes
checking whether to enable assertions... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating config.h
config.status: executing depfiles commands
configure: WARNING: unrecognized options: --disable-rpath, --disable-rpath-hack, --disable-nls, --enable-shared, --enable-static
make  all-recursive
make[1]: Entering directory '/home/builder/.termux-build/jp2a/src'
Making all in src
make[2]: Entering directory '/home/builder/.termux-build/jp2a/src/src'
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o aspect_ratio.o aspect_ratio.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o html.o html.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o terminal.o terminal.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o curl.o curl.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o jp2a.o jp2a.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o options.o options.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I..  -I../include -I/data/data/com.termux/files/usr/include -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -c -o image.o image.c
In file included from jp2a.c:23:
../include/image.h:31:10: fatal error: 'webp/decode.h' file not found
   31 | #include "webp/decode.h"
      |          ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [Makefile:389: jp2a.o] Error 1
make[2]: *** Waiting for unfinished jobs....
image.c:24:10: fatal error: 'webp/decode.h' file not found
   24 | #include "webp/decode.h"
      |          ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [Makefile:389: image.o] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/jp2a/src/src'
make[1]: *** [Makefile:508: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/jp2a/src'
make: *** [Makefile:364: all] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-09-22 18:13:09 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10983283683">10983283683</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/jp2a/build.sh
+++ b/packages/jp2a/build.sh
@@ -1,13 +1,16 @@
-TERMUX_PKG_HOMEPAGE=https://csl.name/jp2a/
+TERMUX_PKG_HOMEPAGE=https://github.com/Talinx/jp2a/
 TERMUX_PKG_DESCRIPTION="A simple JPEG to ASCII converter"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.2.0"
+TERMUX_PKG_VERSION="1.3.0"
 TERMUX_PKG_SRCURL=https://github.com/Talinx/jp2a/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=a19088bb4ba27e8a5524f26f0f1622529ca087e36f5a060c6535c11b266106d1
+TERMUX_PKG_SHA256=68b9cefc291a9bc5b6210dc3cb29173e7222e49a3aec6cab51b0f5e5e696371f
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libcurl, libjpeg-turbo, libpng, ncurses"
+TERMUX_PKG_DEPENDS="libcurl, libexif, libjpeg-turbo, libpng, libwebp, ncurses"
 TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+bashcompdir=${TERMUX_PREFIX}/share/bash-completion/completions
+"
 
 termux_step_pre_configure() {
 	autoreconf -vi

